packages/calendar/src/LionCalendar.js

```diff
@@ -130,11 +130,13 @@ export class LionCalendar extends LocalizeMixin(LitElement) {
 
       /**
        * The currently focused date (if any)
+       * @private
```

Comment:
I don't believe TS looks at static get properties for class fields, so these @private type annotations probably need to be moved to where the class field is initialised (usually in the constructor).